gradle.properties

```diff
@@ -6,23 +6,22 @@
 # http://www.gradle.org/docs/current/userguide/build_environment.html
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
```

Comment:
Should comments be removed?

Reply:
I don't see a reason to remove them.

Reply:
Two reasons:

- Inconsistent with other repos
- Inconsistent with itself, as some params are and some arent commented.

Reply:
This is not within the scope of this PR. The only thing that was needed was to remove the version property.